zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -14,6 +15,21 @@ object HttpData {
   final case class CompleteData(data: Chunk[Byte])             extends HttpData[Any, Nothing]
   final case class StreamData[R, E](data: ZStream[R, E, Byte]) extends HttpData[R, E]
 
+  sealed trait FormDataContent extends Product with Serializable
+
+  final case class FileData(name: String, contentType: String, content: ZStream[Blocking, Throwable, Byte])
+      extends FormDataContent
+  final case class AttributeData(name: String, content: ZStream[Blocking, Throwable, Byte]) extends FormDataContent
```

Comment:
Is there a better way building a stream without depending on `Blocking`?

Reply:
Couldn't find one because [we're doing a `ZStream.fromFile`](https://github.com/dream11/zio-http/blob/99f61ab80488c458a25462166fc2507323ca5d93/zio-http/src/main/scala/zhttp/service/DecodeJRequest.scala#L54) when decoding the multipart. But please let me know if we can provide the `R` earlier already.